src/compile/data/transforms.ts

```diff
@@ -45,6 +48,48 @@ export class FilterNode extends DataFlowNode {
   }
 }
 
+export class GeoJSONNode extends DataFlowNode {
+  public clone() {
+    return new GeoJSONNode(duplicate(this.fields), this.geojson, this.signal);
+  }
+
+  public static make(model: ModelWithField): GeoJSONNode {
+    const geo = model.reduceFieldDef((rel, fieldDef, channel) => {
+      if ((contains([X, Y], channel) && contains([LATITUDE, LONGITUDE], fieldDef.type))
```

Comment:
Wait--I misread your original question. I think the answer to your original question is "no." It generates whichever is applicable to the channel for fieldDef being considered...